lib/gerry/client/groups.rb

```diff
@@ -13,10 +13,17 @@ def groups
 
       # Get all members for a group
       #
+      # @param [Array] options the query parameters
       # @return [Array] the members
-      def group_members(group_id)
+      def group_members(group_id, options = [])
```

Comment:
Currently indeed no other options are documented. However, as more options might be introduced in the future, I decided to follow the existing pattern in [accounts.rb](https://github.com/trumant/gerry/blob/master/lib/gerry/client/accounts.rb#L8) and [changes.rb](https://github.com/trumant/gerry/blob/master/lib/gerry/client/changes.rb#L8). Still I agree that your proposal makes the method slightly more convenient to use right now. If you still want me to change it I'll do so.